.github/workflows/ci_tests_dev.yaml

```
@@ -125,7 +129,7 @@ jobs:
           cd build
           call "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
           cmake -G Ninja .. ^
-            -DCMAKE_INSTALL_PREFIX=${{ env.GMT_INSTALL_DIR }} ^
+            -DCMAKE_INSTALL_PREFIX=${env:GMT_INSTALL_DIR} ^
```

The Windows CI is failing, maybe try the one below (xref: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#using-the-env-context-to-access-environment-variable-values):
```suggestion
            -DCMAKE_INSTALL_PREFIX=$env:GMT_INSTALL_DIR ^
```

---

Don't think it's because of the path, it looks set to `GMT_INSTALL_DIR: D:\a\_temp/gmt-install-dir` with or without the curly braces, same as at https://github.com/GenericMappingTools/pygmt/actions/runs/13889360104/job/38858639159#step:8:21

---

Owh, looks like the Cmake build on Windows didn't complete - https://github.com/GenericMappingTools/pygmt/actions/runs/13934908307/job/39000590987#step:8:669

```bash
[0/1] Install the project...
CMake Warning (dev) at cmake_install.cmake:5 (set):
  Syntax error in cmake code at

    D:/a/pygmt/pygmt/gmt/build/cmake_install.cmake:5

  when parsing string

    D:/a/pygmt/pygmt/gmt/build/${env:GMT_INSTALL_DIR}

  syntax error, unexpected cal_SYMBOL, expecting } (30)

  Policy CMP0010 is not set: Bad variable reference syntax is an error.  Run
  "cmake --help-policy CMP0010" for policy details.  Use the cmake_policy
  command to set the policy and suppress this warning.
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Install configuration: "Release"
CMake Error at src/cmake_install.cmake:36 (file):
  file cannot create directory:
  D:/a/pygmt/pygmt/gmt/build/${env:GMT_INSTALL_DIR}/lib.  Maybe need
  administrative privileges.
Call Stack (most recent call first):
  cmake_install.cmake:37 (include)


FAILED: CMakeFiles/install.util 
C:\Windows\system32\cmd.exe /C "cd /D D:\a\pygmt\pygmt\gmt\build && "C:\Program Files\CMake\bin\cmake.exe" -P cmake_install.cmake"
ninja: build stopped: subcommand failed.
```

---

Yeah, looking at https://github.com/GenericMappingTools/pygmt/actions/runs/13934908307/job/39000590987#step:8:435, there are these lines:

```
*  Locations:
*  Installing GMT in          : D:/a/pygmt/pygmt/gmt/build/${env:GMT_INSTALL_DIR}
*  GMT_DATADIR                : D:/a/pygmt/pygmt/gmt/build/${env:GMT_INSTALL_DIR}/share
*  GMT_DOCDIR                 : D:/a/pygmt/pygmt/gmt/build/${env:GMT_INSTALL_DIR}/share/doc
*  GMT_MANDIR                 : D:/a/pygmt/pygmt/gmt/build/${env:GMT_INSTALL_DIR}/share/man
```

So the environment variable replacement isn't working on powershell

---

Ok, syntax should be `%GMT_INSTALL_DIR%` according to https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-running-a-command-using-windows-cmd :crossed_fingers: